templates/web/base/admin/update_edit.html

```diff
@@ -55,10 +55,42 @@
 </li>
 <li>[% loc('Cobrand:') %] [% update.cobrand %]</li>
 <li>[% loc('Cobrand data:') %] [% update.cobrand_data OR loc('None') %]</li>
+[% IF update.external_id~%]
+  <li>[% loc('External ID:') %] [% update.external_id %] </li>
```

Comment:
To save on translation, note how "External ID" is already in the .po file, but "External ID:" is not. So if you had `[% loc('External ID') %]:` (which matches the other uses) it could share the translation.